README.md

```diff
@@ -9,4 +9,93 @@ pages and provided as a reference for recipe implementation
 It is recommended for use during bring-up and that you implement only
 required components for deployment in production environments
 
-use "-h" for more details
\ No newline at end of file
+use "-h" for more details
+
+
+# nv-mlnxipcfg.py
+
+nv-mlnxipcfg.py configures and sets persistency behavior for IP addresses, IP rules, IP route, and ARP settings
+
+Requires: Python 3.x and python3-click
```

Comment:
This is my biggest concern - that there are non-inbox (I assume) dependencies for this code to run without a requirements.txt file or something similar to facilitate someone bootstrapping this correctly. Is it too much trouble to ask that you include a requirements.txt file, along with instructions on how to use it?